.kitchen.yml

```diff
@@ -12,18 +13,19 @@ verifier:
   name: inspec
 
 platforms:
-- name: amazon-linux
-  driver_config:
-    box: mvbcoding/awslinux
-- name: centos-6
-- name: centos-7
-- name: debian-8
-- name: debian-9
-- name: fedora-27
-- name: opensuse-leap-42
-- name: ubuntu-14.04
-- name: ubuntu-16.04
+  - name: amazon-linux
+    driver_config:
+      box: mvbcoding/awslinux
+  - name: centos-6
+  - name: centos-7
+  - name: debian-8
+  - name: debian-9
+  - name: fedora-27
+  - name: opensuse-leap-42
+  - name: ubuntu-14.04
+  - name: ubuntu-16.04
```

Comment:
I think this list is now "wrong" 

it could do with 
- adding amazonlinux-2 
- removing 14.04

whatever you want with centos-6